README.rst

```diff
@@ -314,6 +314,23 @@ Code utilities
 
 Generates an random string of 32 characters.
 
+``openwisp_utils.utils.add_default_menu_items``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Takes input of a list of models name and settings_menu name and adds
+them to the side navigation bar in the admin dashboard interface.
+
+Usage:
+
+.. code-block:: python
+
+    from openwisp_utils.utils import add_menu_items
```

Comment:
would be good to suggest to use this in the `ready()` method of an `AppConfig` class.